blosc/blosc2.c

```diff
@@ -3398,6 +3401,10 @@ void blosc_init(void) {
   /* Return if Blosc is already initialized */
   if (g_initlib) return;
 
+#if defined(HAVE_PLUGINS)
```

Comment:
@FrancescAlted , Wouldn't it be better if the plugins imports were done here?

Reply:
Yes, that makes sense to me.